### PR TITLE
Synthesizing tasks for IS 210 Week Two

### PR DESCRIPTION
**Pull request by clever-username**
Completed all tasks, ran unit and lint tests w/o issue.

### CODE REVIEW

**Comment by clever-username** [comment]
Failed to run test_task_03.py before submitting pull request; changes made and pushed.

File: task_01.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+    me@mymachine $ python my_file_name.py
+
+**********
+The Syntax
+**********
+
+Python syntax is very easy to write. Take a look at this simple statement:
+    
+.. code:: python
```
Comment (cheuschober) [inline task_01.rst]: Just nitpicking here for your own edfication, the general python code directive here is good but this specific case is of the python console or python interpreter in which case the `pycon` lexer is the best choice, eg:

    .. code:: pycon

You can see the full list of available lexers at: http://pygments.org/docs/lexers/

File: task_01.rst [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+   ====== ===============
+   Name   Contact        
+   ====== ===============
+   Me     me@mysite.com
+   Myself myself@me.com  
+   I      i@mii.com
+   ====== ===============
+   
+.. Note::
+    This is just a little taste of how great Python can be!
```
Comment (cheuschober) [inline task_01.rst]: Generally, I recommend a full blank space between any directives and their text bodies so line 96, ideally, would be a blank space with your text starting on 97 just as you did with 85-87 of your table.

File: task_03.py [hunk 3]
```diff
@@ -1,7 +1,8 @@
-CHARLIE = "Chocolate Factory" 
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""A simple Python file."""
+
+CHARLIE = "Brown"
```
Comment (cheuschober) [inline task_03.py]: Good catch of the unit test!

**Pull request merged**